projects/workflows-creator/src/lib/builder/tooltip-render/tooltip-render.component.ts

```diff
@@ -5,13 +5,11 @@ import {Component, Input, OnInit} from '@angular/core';
   templateUrl: './tooltip-render.component.html',
   styleUrls: ['./tooltip-render.component.scss'],
 })
-export class TooltipRenderComponent implements OnInit {
+export class TooltipRenderComponent {
```

Comment:
??

Reply:
it gives error of linit empty onInit thats why deleted